monkey/common/types.py

```diff
@@ -29,16 +41,16 @@ class PortStatus(Enum):
     An Enum representing the status of the port.
 
     This Enum represents the status of a network pork. The value of each
-    member is distincive and unique number.
+    member is the member's name in all lower-case characters.
     """
 
-    OPEN = 1
-    CLOSED = 2
+    OPEN = "open"
```

Comment:
What prompted this change? Readability of serialized data?

Reply:
Yep, readability of serialized data.